services/telemetry/telemetry.go

```diff
@@ -183,6 +184,19 @@ func (ts *TelemetryService) sendTelemetry(event string, properties map[string]in
 }
 
 func isDefault(setting interface{}, defaultValue interface{}) bool {
+	if reflect.TypeOf(setting).Kind() == reflect.Slice && reflect.TypeOf(defaultValue).Kind() == reflect.Slice {
```

Comment:
Oh sorry for any confusion, I just meant to go with our immediate case of slice of strings and work with that. Importing the reflect package is effectively the same as using `reflect.DeepEqual` itself. If we see ourselves using slices of other things as well, we can use `reflect.DeepEqual` then.
```go
func comp(setting, defaultValue interface{}) bool {
	slice1, ok1 := setting.([]string)
	slice2, ok2 := defaultValue.([]string)
	if ok1 && ok2 {
		if len(slice1) != len(slice2) {
			return false
		}
		for i := 0; i < len(slice1); i++ {
			if slice1[i] != slice2[i] {
				return false
			}
		}
		return true
	}
	return setting == defaultValue
}
```

Apologies for the double work.

Reply:
Understood! I think you have explained it right but my limited exposure to golang is to blame for this 😄 I thought that's the only available way to check the type and get the value. Thanks for the review and help! 🙂

Reply:
Also I was trying to cover int slices as well (that's not in the use case but thought when we have it will be covered automatically). I guess that's where I went in the wrong direction into using reflect package again 😅